src/js/disability-benefits/utils/appeal-helpers.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+export const appealStatusDescriptions = (eventType) => {
```

Comment:
I thought they would. You're still using them inside an object in the function. I don't know if wrapping them within a function is a requirement for React nodes. But I'm not opposed to doing it either way.